src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2170,61 +2170,72 @@ export class StudyViewPageStore
 
     @computed
     get initialFilters(): StudyViewFilter {
-        let initialFilter = {} as StudyViewFilter;
-        if (_.isEmpty(this.queriedSampleIdentifiers.result)) {
-            initialFilter.studyIds = this.queriedPhysicalStudyIds.result;
-        } else {
-            initialFilter.sampleIdentifiers = this.queriedSampleIdentifiers.result;
+        if (!this.initialFiltersRemote.isComplete) {
```

Comment:
is this still necessary after change that we made for the hesitate PR, now merged?